frontend/src/screens/learner.tsx

```diff
@@ -51,7 +51,7 @@ const StudyList: FCWOC<StudyListProps> = ({ className, onSelect, title, studies,
             {children}
             {!studies.length && <h3>Awesome, you completed all studies! Watch out for new studies coming up soon!</h3>}
             <Grid css={{ overflow: 'auto', paddingBottom: '10px' }} data-test-id="studies-listing">
-                {studies.map(s => <StudyCard onSelect={onSelect} study={s} key={s.id} />)}
+                {studies.map((s, i) => <StudyCard onSelect={onSelect} study={s} key={i} />)}
             </Grid>
```

Comment:
if possible, it's preferable to use something other than the index for a key. https://articles.wesionary.team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9